debezium-testing/debezium-testing-system/src/main/java/io/debezium/testing/system/tools/OpenShiftUtils.java

```diff
@@ -260,6 +262,27 @@ public void waitForPods(String project, Map<String, String> labels) {
         }
     }
 
+    public void deletePodsOfDeployment(Deployment deployment) {
```

Comment:
This is not what the method does and if it were then the implementation would be highly unreliable.  The original log message was a bit misleading. 

The method scales the deployment to 0... the "waiting for pods to delete" is there to actually "wait before the deployment is scaled to 0".